Sources/Error.swift

```diff
@@ -12,12 +12,21 @@ protocol ANSIConvertible : Error, CustomStringConvertible {
 
 extension ANSIConvertible {
   func print() {
-    if isatty(fileno(stderr)) != 0 {
+    // Check if we are in any term env and the output is a tty.
+    let termType = getEnvValue("TERM")
+    if let t = termType, t.lowercased() != "dumb" && isatty(fileno(stdout)) != 0 {
       fputs("\(ansiDescription)\n", stderr)
     } else {
       fputs("\(description)\n", stderr)
     }
   }
+
+  private func getEnvValue(_ key: String) -> String? {
+    guard let value = getenv(key) else {
+      return nil
+    }
+    return String(cString: value)
+  }
```

Comment:
Nitpicking, but not sure it's worth having that private function anymore since `XcodeColors` isn't user anymore.

So using this directly for line 16 would suffice 😛 
```swift
let termType = getenv("TERM").flatMap({ String(cString: $0) })
```